account_analytic_sequence/i18n/account_analytic_sequence.pot

```diff
@@ -0,0 +1,29 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* account_analytic_sequence
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0\n"
```

Comment:
I see in the PO files the version is wrong (it's 10.0 in most of them). I don't know if we have to keep them because it indicates the version in which the translations were first added. However in the POT, I should change it to the current version 17.0.